### ISSUE TITLE
Cannot backup Paused VM, virt freezer fails:

### ISSUE THREAD

**Issue by brybacki** [kubevirt/kubevirt-velero-plugin#68]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind enhancement

**What happened**:

Found in `VM+VMI included, Pod excluded: should succeed if VM is paused` 

Cannot backup Paused VM, virt freezer fails:

```
time="2022-06-03T13:51:37Z" level=info msg="Backing up item" backup=velero/test-backup-1654264228561761698 logSource="pkg/backup/item_backupper.go:121" name=virt-launcher-test-vm-6dr85 namespace=kvp-e2e-tests-bpf2z resource=pods
time="2022-06-03T13:51:37Z" level=info msg="running exec hook" backup=velero/test-backup-1654264228561761698 hookCommand="[/usr/bin/virt-freezer --freeze --name test-vm --namespace kvp-e2e-tests-bpf2z]" hookContainer=compute hookName="<from-annotation>" hookOnError=Fail hookPhase=pre hookSource=annotation hookTimeout="{30s}" hookType=exec logSource="pkg/podexec/pod_command_executor.go:124" name=virt-launcher-test-vm-6dr85 namespace=kvp-e2e-tests-bpf2z resource=pods
time="2022-06-03T13:51:37Z" level=info msg="stdout: " backup=velero/test-backup-1654264228561761698 hookCommand="[/usr/bin/virt-freezer --freeze --name test-vm --namespace kvp-e2e-tests-bpf2z]" hookContainer=compute hookName="<from-annotation>" hookOnError=Fail hookPhase=pre hookSource=annotation hookTimeout="{30s}" hookType=exec logSource="pkg/podexec/pod_command_executor.go:171" name=virt-launcher-test-vm-6dr85 namespace=kvp-e2e-tests-bpf2z resource=pods
time="2022-06-03T13:51:37Z" level=info msg="stderr: {\"component\":\"freezer\",\"level\":\"info\",\"msg\":\"Starting...\",\"pos\":\"main.go:46\",\"timestamp\":\"2022-06-03T13:51:37.954237Z\"}\n{\"component\":\"freezer\",\"level\":\"error\",\"msg\":\"Freezeing VMI failed\",\"pos\":\"main.go:81\",\"reason\":\"server error. command Freeze failed: \\\"LibvirtError(Code=55, Domain=10, Message='Requested operation is not valid: domain is not running')\\\"\",\"timestamp\":\"2022-06-03T13:51:37.965276Z\"}\n" backup=velero/test-backup-1654264228561761698 hookCommand="[/usr/bin/virt-freezer --freeze --name test-vm --namespace kvp-e2e-tests-bpf2z]" hookContainer=compute hookName="<from-annotation>" hookOnError=Fail hookPhase=pre hookSource=annotation hookTimeout="{30s}" hookType=exec logSource="pkg/podexec/pod_command_executor.go:172" name=virt-launcher-test-vm-6dr85 namespace=kvp-e2e-tests-bpf2z resource=pods
time="2022-06-03T13:51:37Z" level=error msg="Error executing hook" backup=velero/test-backup-1654264228561761698 error="command terminated with exit code 1" hookPhase=pre hookSource=annotation hookType=exec logSource="internal/hook/item_hook_handler.go:206" name=virt-launcher-test-vm-6dr85 namespace=kvp-e2e-tests-bpf2z resource=pods
time="2022-06-03T13:51:37Z" level=error msg="Error backing up item" backup=velero/test-backup-1654264228561761698 error="command terminated with exit code 1" logSource="pkg/backup/backup.go:441" name=virt-launcher-test-vm-6dr85
```

**What you expected to happen**:

Backup should succeed

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- CDI version (use `kubectl get deployments cdi-deployment -o yaml`):
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Install tools:
- Others:

**Comment by brybacki** [comment]
https://github.com/kubevirt/kubevirt/issues/7750

**Comment by brybacki** [comment]
fixed in kubevirt 0.57, should work after updating
